pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringservicelevelobjective/windowbasedmetricsumfilter/update.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     external: projects/${projectId}
   serviceRef:
     external: monitoringservice-${uniqueId}
-  displayName: "updae request - a window based metric sum filter"
+  displayName: "update request - a window based metric sum filter"
```

Comment:
This will break this test. To fix that, could you run `hack/compare-mock fixtures/windowbasedmetricsumfilter` ? Or just drop this change. Ditto to the pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringservicelevelobjective/windowbasedmetricsumfilter/_generated_object_windowbasedmetricsumfilter.golden.yaml file above

Reply:
I think https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3218/commits/92d726eea88a3f6a43987fb90922e7a2bb787e48 does what you want